contracts/Vault.vy

```diff
@@ -954,13 +954,19 @@ def _reportLoss(strategy: address, loss: uint256):
     # Loss can only be up the amount of debt issued to strategy
     totalDebt: uint256 = self.strategies[strategy].totalDebt
     assert totalDebt >= loss
+
+    # Also, make sure we reduce our trust with the strategy by the amount of loss
+    ratio_change: uint256 = 0
+    if self.debtRatio != 0: # if vault with single strategy that is set to EmergencyOne
+        ratio_change = min(
+            # NOTE: This calculation isn't 100% precise, the adjustment is ~10%-20% more severe due to EVM math
+            MAX_BPS * loss / (self.totalDebt * MAX_BPS / self.debtRatio),
```

Comment:
We can actually cancel both uses of `MAX_BPS`, and rearrange terms as follows
```suggestion
            loss * self.debtRatio / self.totalDebt,
```

NOTE: use "reciprocal of reciprocal" rule
![Screenshot_20210513-092603_Brave.jpg](https://user-images.githubusercontent.com/3859395/118132623-e49b9080-b3cd-11eb-85b3-5011b071717d.jpg)